### ISSUE TITLE
Could order and cart templates be combined?

### ISSUE THREAD

**Issue by racitup** [awesto/django-shop#631]
The order detail and cart static templates are very similar, could they be combined?

As a minor comment, I also prefer the column order to be:
**Product, Quantity, Unit Price, Total** rather than **Quantity, Product, Unit Price, Total** because it is quantity * unit_price = total, so in my opinion those three fields belong closer together.

**Comment by jrief** [comment]
You're right, sometimes they are very similar, but sometimes not. I'd rather prefer to reuse the existing order template and just ``{% include "path/to/cart.html" %}``. I believe that this is more explicit.

Agree, the column ordering: Quantity, Product, Unit Price, Total is more appropriate.
